### DISCUSSION TITLE
Can't change stroke-width on Vue icon components

### DISCUSSION THREAD

**Discussion by Jacobtims** [tailwindlabs/heroicons#946] [Help]
I want to change the stroke-width of an icon on the Vue component.
Like this:
```
import { PlusIcon } from '@heroicons/vue/24/solid';

<PlusIcon class="h-5 w-5" stroke-width="2"/>
```

This unfortunately **doesn't work**. 
How can I change the stroke-width?

**Comment by ruffjaxson** [comment]
I have this problem but in React
